avalanche/training/templates/base.py

```diff
@@ -106,7 +106,11 @@ def train(
             experiences = [experiences]
         if eval_streams is None:
             eval_streams = [experiences]
-        self._eval_streams = _group_experiences_by_stream(eval_streams)
+
+        if len(eval_streams) == 1:
+            self._eval_streams = eval_streams
+        else:
```

Comment:
can you put this inside`_group_experiences_by_stream`?

Reply:
yea, sure.